config/routes.rb

```diff
@@ -4,9 +4,16 @@
   mount Sidekiq::Web, at: '/sidekiq' if defined? Sidekiq::Web
   mount Karafka::Web::App, at: '/karafka' if ENV['KARAFKA_WEB']
 
-  mount GraphiQL::Rails::Engine, at: '/graphiql', graphql_path: '/graphql' if Rails.env.development?
+  if Rails.env.development?
+    mount GraphiQL::Rails::Engine, at: '/customer_portal_graphiql', as: 'customer_portal_graphiql_rails', graphql_path: '/customer_portal_graphql'
+    mount GraphiQL::Rails::Engine, at: '/api_graphiql', as: 'api_graphiql_rails', graphql_path: '/api_graphql'
+  end
+
+  # TODO: Remove this deprecated route.
+  post '/graphql', to: 'graphql/api#execute'
```

Comment:
this change will require sync with FE to update customer portal requests, isn't it?

The current FE implementation will send request to this graphQL endpoint, and will be routed to the API namespace queries/mutations. Then, with the changes in this PR, as an admin accessing the customer portal will work as you have the GraphQL cookie with your privileged admin user access. However, I guess as a real customer accessing the portal with the url (incognito windows) it will fail as the new API controller does not handle the customer portal user variable.

This is related to the previous comments in the query and mutation types for the API namespace.